src/components/App/App.tsx

```diff
@@ -41,6 +43,11 @@ export function App() {
         hideLabel={false}
         helperText="this is helper text"
       />
+      <Tabs currentTab={currentTab} onTab={setCurrentTab}>
```

Comment:
I think it might make more sense to expose an `id` prop on the Tab component and use the id for handling active states. This would also provide the user with a more meaningful value that would be returned from the onTab callback. I'm just thinking in terms of routing/api requests. Even outside of those considerations, I think reasoning through a value that is returned that they can see assigned to a given tab makes more sense than an index, which feels a little more arbitrary.

Reply:
@thispastwinter yeah, I think that's fair. I had considered just passing the entire tab data back, which would give the user everything. I do like the idea of requiring an `id` prop on `Tab` and using it throughout. I think that's smart. Great suggestion!